.circleci/config.yml

```diff
@@ -89,6 +89,22 @@ jobs:
       - run:
           name: Update vresion.rb and publish on RubyGems
           command: bash .circleci/deploy_to_rubygems.sh
+  publish_to_docker_hub:
+    machine: true
+    environment:
+      DOCKER_REPO: lifullsetg/bucky-core
+    steps:
+      - checkout
+      - run:
+          name: Build image
+          command: |
+            docker build -t "$DOCKER_REPO":"${CIRCLE_TAG/v/}" .
+      - run:
+          name: Login and push image
+          command: |
+            docker login -u "$DOCKER_HUB_USER" -p "$DOCKER_HUB_TOKEN"
```

Comment:
Could you tell me where These Variables are stored?
I couldn't find in liufll-dev`s Circle CI Organization.

Reply:
I haven't stored these variables yet.

Reply:
I set.